pkg/utils/sentry/trace.go

```diff
@@ -28,15 +29,38 @@ func Init(conf Config) (err error) {
 	return nil
 }
 
-// Middleware that tracks request performance with Sentry.
-func TrackPerformance() gin.HandlerFunc {
+// Gin middleware that tracks HTTP request performance with Sentry.
+func TrackPerformance(tags map[string]string) gin.HandlerFunc {
 	return func(c *gin.Context) {
-		span := sentry.StartSpan(c.Request.Context(), c.Request.URL.Path)
+		request := fmt.Sprintf("%s %s", c.Request.Method, c.Request.URL.Path)
+		span := sentry.StartSpan(c.Request.Context(), "http handler", sentry.TransactionName(request))
```

Comment:
Similarly - to distinguish between the API types name the operations `grpc` and `rest`?

```suggestion
		span := sentry.StartSpan(c.Request.Context(), "rest", sentry.TransactionName(request))
```